### PR TITLE
Event

### PR DESCRIPTION
**Pull request by Shahvinit3113**

### CODE REVIEW

File: src/api/EventApi.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            toast.error(response.message);
+        }
+        return response
+    } catch (error) {
+        console.error(error)
+    }
+}
+
+export const AddImageApi = async (eventid, img, handleClose, handleSaveChanges) => {
+    console.log(eventid, 'key');
```
Comment (pooja-vipulbhai-darji) [inline src/api/EventApi.js:28]: remove this

File: src/pages/Dashboard/Event.js [hunk 2]
```diff
@@ -38,35 +89,187 @@ function Event() {
...
+              </div>
+            </Box>
+          </Modal>
         </div>
+        <BaseTable tableName='Event register' searchLabel={dashboard.search} searchPlaceholder={dashboard.searchHere} columns={columns} onSearch={handleSearch} searchTerm={searchTerm} Data={filteredEvents?.map((value) => ({
+          ...value,
+          event_manage_images: (
+            <img
+              src={`${imageURL}/${value.event_manage_images?.[0]?.image}`}
+              style={{ height: "80px", width: "80px", objectFit: "cover", margin: "0px" }}
```
Comment (pooja-vipulbhai-darji) [inline src/pages/Dashboard/Event.js:244]: remove this inline style

File: src/pages/Dashboard/Event.js [hunk 3]
```diff
@@ -3,21 +3,72 @@ import { DashboardEventApi } from '../../api/DashboardApi';
...
+import BaseInput from '../../base/BaseInput'
+import { useFormik } from 'formik';
+import { EventAddSchema, EventEditSchema } from '../../schema/EventSchema';
+import { AddEventApi, AddImageApi, deleteEventApi, editEventApi } from '../../api/EventApi';
+import { Typography } from '@mui/material';
+import { dashboard, events, profile } from '../../constants';
+import BaseLoader from '../../base/BaseLoader'
+
+const style = {
+  position: 'absolute',
```
Comment (pooja-vipulbhai-darji) [inline src/pages/Dashboard/Event.js:17]: why add this type of style

File: src/pages/Dashboard/Event.js [hunk 4]
```diff
@@ -38,35 +89,187 @@ function Event() {
...
+      event_name: "",
+      event_description: "",
+      profile_image: null,
+    },
+    validationSchema: EventAddSchema,
+    onSubmit: async (values) => {
+      setIsLoading(true)
+      try {
+        await AddEventApi(values).then(response => {
+          if (response.status === 'success') {
```
Comment (pooja-vipulbhai-darji) [inline src/pages/Dashboard/Event.js:126]: Use the Status code

File: src/pages/Dashboard/Event.js (hunk 3 above)
Comment (Shahvinit3113) [reply src/pages/Dashboard/Event.js:17]: Replaced with normal one

**Pull request merged**